projects/ades1754/README.rst

```diff
@@ -0,0 +1,244 @@
+	|  VCC_3V3  |   Connect to DCIN2 pin of MAX17851EVKIT	      |
+	+-----------+-------------------------------------------------+
+	|  VCC_5V   |   Connect to DCIN1 pin of MAX17851EVKIT	      |
+	+-----------+-------------------------------------------------+
+	|  GND      |   Connect to GND pin of MAX17851EVKIT	      |
+	+-----------+-------------------------------------------------+
+	|  CS	    |   Connect to SPI Chip Select(CS) 		      |
+	+-----------+-------------------------------------------------+
+	|  MOSI	    |   Connect to SPI Master-Out-Slave-In(MOSI)      |
+	+-----------+-------------------------------------------------+
+	|  MISO	    |   Connect to SPI Master-In-Slave-Out(MISO)      |
+	+-----------+-------------------------------------------------+
+	|  SCK	    |   Connect to SPI Clock(SCK) 	      	      |
+	+-----------+-------------------------------------------------+
+
+	+-----------+-------------------------------------------------+
+	|   Name    |	Description				      |
+	+-----------+-------------------------------------------------+
+	|  VCC_3V3  |   Connect to SHDNL pin			      |
+	+-----------+-------------------------------------------------+
+	|  GND      |   Connect to AGND pin			      |
+	+-----------+-------------------------------------------------+
+	|  RXN      |   Connect to TXN signal of MAX17851EVKIT	      |
+	+-----------+-------------------------------------------------+
+	|  RXP	    |   Connect to TXP signal of MAX17851EVKIT	      |
+	+-----------+-------------------------------------------------+
+	|  TXN	    |   Connect to RXN signal of MAX17851EVKIT        |
+	+-----------+-------------------------------------------------+
+	|  TXP	    |   Connect to RXP signal of MAX17851EVKIT	      |
+	+-----------+-------------------------------------------------+
+
+No-OS Build Setup
+-----------------
+
+Please see: `No-OS Build Wiki <https://wiki.analog.com/resources/no-os/build>`_
+
+No-OS Supported Examples
+------------------------
+
+The initialization data used in the examples is taken out from:
+`Project Common Data Path <https://github.com/analogdevicesinc/no-OS/tree/main/projects/ades1754/src/common>`_
+
+The macros used for Common Data structures are defined in pkatform specific files found at:
+`Project Platform Configuration Path <https://github.com/analogdevicesinc/no-OS/tree/main/projects/ades1754/src/platform>`_
+
+DEF UART Example
+^^^^^^^^^^^^^^^^
+
+Steps of the examples:
+
+* Initializes the ADES1754
+* Reads Cell Voltage for each Cell.
+* Scans for Alerts.
+* In case any alerts are detected the example is stopped.
+* If the example is stopped a specific message is shown on the Std I/O UART.
+
+In order to build the default uart example make sure you have the following
+configuration in the Makefile:
+`Makefile <https://github.com/analogdevicesinc/no-OS/tree/main/projects/ades1754/Makefile>`_
+
+.. code-block:: bash
+
+	EXAMPLE ?= def_uart_example
+
+SPI UART Example
```

Comment:
really, this example name should be more telling, `SPI UART` doesnt make sense to me, but `SPI UART bridge` does